.golangci.yml

```diff
@@ -18,7 +18,12 @@ linters:
     - unparam # allow constant parameters
     - goerr113 # allow "dynamic" errors
     - dogsled # allow blank identifiers
-    - gocyclo # allow complex functions
+    - cyclop # allow complex functions
     - nlreturn # allow return/break without whitespace
     - exhaustive # TODO: make switches exhaustive and enable
     - noctx # TODO: update http calls and enable
+    - wrapcheck # TODO: wrap errors and enable
```

Comment:
I'd just keep this off to be honest. Explicitly exposing every error without giving it some thought on each is a bad idea specially in an open source repo.

Reply:
true